HorribleDownloader/cmd.py

```diff
@@ -224,12 +224,15 @@ def should_download(episode):
                 if inp not in ('', 'Y', 'y', 'yes', 'Yes'):
                     print(fg(1) + 'aborting download\n' + fg.rs)
                     exit(1)
-
-        for episode in filtered_episodes:
-            download(episode, qualities, config.download_dir)
+                          
+        if args.export:
```

Comment:
You must first add an argument for export, it must have an understandable description

Reply:
My version has this, I was quick and did the edit manually again here on the web interface and missed that line, I will correct.